### PR TITLE
🐛 Fixing regular expression of `[...]` used in European Portuguese Hunspell dictionary

### PR DESCRIPTION
**Pull request by gusbemacbe**
Hello @yzhang-gh 

It is ready! I tested and it worked. The `[...]` has been successfully removed from the autocompletion pop-up when searching a European Portuguese word.

### CODE REVIEW

File: src/completion.ts [hunk 1]
```diff
@@ -132,7 +132,7 @@ function loadOtherWordsAndRebuildIndex(builtInWords: string[]) {
                     if (/\d+/.test(list[0])) {
                         list.splice(0, 1);
                     }
-                    list = list.map(word => word.replace(/\/.*$/, ''));
+                    list = list.map(word => word.replace(/\/.*$|\[.*$/, ''));
```
Comment (yzhang-gh) [inline src/completion.ts:135]: It can be `/[\/\[].*$/`

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "displayName": "Dictionary Completion",
     "description": "Word Completion",
     "icon": "images/dictionary.png",
-    "version": "1.2.2",
+    "version": "1.2.3",
```
Comment (yzhang-gh) [inline package.json:6]: Can you revert the changes to `package.json` and `package-lock.json`? I usually keep them in a separate commit.

File: src/completion.ts (hunk 1 above)
Comment (gusbemacbe) [reply src/completion.ts]: Done.

File: package.json (hunk 2 above)
Comment (gusbemacbe) [reply package.json]: Done.

**Comment by gusbemacbe** [comment]
Done!

**Comment by yzhang-gh** [comment]
Thanks!

You can try it out using the [dev build](https://github.com/yzhang-gh/vscode-dic-completion/actions/runs/3723418393) (Artifacts) right now.

**Comment by gusbemacbe** [comment]
I have downloaded and tested it, and it worked, @yzhang-gh
